docs/source/sphinxext/gallery_generator.py

```diff
@@ -18,6 +18,9 @@
 from matplotlib import image
 
 DOC_SRC = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+DEFAULT_IMG_LOC = os.path.join(
+            os.path.dirname(DOC_SRC), "logos", "PyMC3.png"
```

Comment:
somehow this is getting way spread out... i'll look more at how to fix it tomorrow...

Reply:
Thanks!